docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -1,5 +1,98 @@
 ---
 apiVersion: v1
+kind: policy
+spec:
+  name: Ensure a password is required to wake the computer from sleep or screen saver is enabled
+  platforms: macOS
+  platform: darwin
+  description: Checks that Password is Required to Wake the Computer From Sleep or Screen Saver Is Enabled.
```

Sorry I know this was probably already like this but I can't ignore the capitalization

```suggestion
  description: Checks that password is required to wake the computer from sleep or screen saver is enabled.
```